ipv-core/src/main/java/com/ibm/whc/deid/util/ReversePatternManager.java

```diff
@@ -101,7 +101,8 @@ public ReversePatternGenerator getPatternByResource(String patternName, String l
   /** Get the collection of all pattern resource locations, for all countries and languages */
   @Override
   protected Collection<ResourceEntry> getResources() {
-    return LocalizationManager.getInstance().getResources(Resource.PATTERN);
+		return LocalizationManager.getInstance(LocalizationManager.DEFAULT_LOCALIZATION_PROPERTIES)
```

Comment:
Should the localizationPropertty from the constructor be used here?  If we allow the constructor to be more visible than "private", its possible a caller would have passed a different property value.